service/src/test/resources/ehr/mapper/non_consultation_bundles/medication-request.xml

```diff
@@ -0,0 +1,52 @@
+<component typeCode="COMP" contextConductionInd="true">
```

Comment:
contextConductionInd should not be present - an output MedicationStatement will never be nested